common/lib/xmodule/xmodule/course_module.py

```diff
@@ -202,6 +206,13 @@ class CourseFields(object):
                     "weight": 0.15,
                 },
                 {
+                    "type": "Entrance Exam",
+                    "short_label": "Entrance",
+                    "min_count": 0,
+                    "drop_count": 0,
+                    "weight": 0.0,
+                },
```

Comment:
I believe this will mean that all newly created courses will have this grading policy? I believe there is code which sets this when the author selects to have an entrance exam. Should we then leave it off of the default value?